### ISSUE TITLE
Wrong indentation for `where` clauses in interfaces in c#

### ISSUE THREAD

**Issue by KalyanaChakravarthi** [uncrustify/uncrustify#1619]
```diff --git a/UNI-36664.cs b/test.out.cs
index 89e2e7eb..794e8a1a 100644
--- a/UNI-36664.cs
+++ b/test.out.cs
@@ -1,3 +1,2 @@
 long DeviceCommand<TCommand>(int deviceId, ref TCommand command)
-    where TCommand : struct, IInputDeviceCommandInfo;

+where TCommand : struct, IInputDeviceCommandInfo;
```
[UNI-36664.cs](https://github.com/uncrustify/uncrustify/files/1727075/UNI-36664.cs.txt)

**Comment by satishp87** [comment]
In unity fork this issue is addressed with below PR
Unity-Technologies#93

**Comment by guy-maurel** [comment]
Fixed at 4f034b5a7827b4b0f7c578bdcada927b4a5c12da, on Dec  6 19:38:38 2017
